src/domain/Avatars/AvatarEntity/style.ts

```diff
@@ -15,20 +16,45 @@ interface IStyledAvatarEntityProps {
 
 interface IAvatarEntityInfoProps {
   isCompact: boolean,
-  size: Size
+  size: AvatarEntitySize
 }
 
+interface IStyledAvatarProps {
+  size: AvatarEntitySize
+}
+
+interface IStyledTertiaryTextProps {
+  size: AvatarEntitySize
+}
+
+// use height in avatar-medium or avatar-small
+const StyledAvatar = styled.span`
+  ${(props: IStyledAvatarProps) => css`
+      height: ${props.size === AvatarEntitySize.Normal ? 40 :  30}px;
+  `}
+`
+
 const StyledAvatarEntity = styled.div`
   ${(props: IStyledAvatarEntityProps) => styleForMargins(props.margins)}
 `
 
+// use height in avatar-medium or avatar-small add Variables.Spacing.sXSmall
```

Comment:
This comment doesn't really make sense to me. I would write something like:
```
// We're using magic numbers here to align the tertiary text with the primary and secondary text because it sits outside of the main avatar entity content 
```

Reply:
The only reason we're doing it like this is so we can have a consistent height for the main content right?

Reply:
It is avatar width plus the spacing for alignment